app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -731,6 +766,74 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Activit
         initializeEngineView(toolbarHeight)
     }
 
+    /**
+     * Shows a biometric prompt and fallback to prompting for the password.
+     */
+    private fun showBiometricPrompt(context: Context) {
+        if (BiometricPromptFeature.canUseFeature(context)) {
+            biometricPromptFeature.get()
+                ?.requestAuthentication(getString(R.string.credit_cards_biometric_prompt_unlock_message))
+            return
+        }
+
+        // Fallback to prompting for password with the KeyguardManager
+        val manager = context.getSystemService<KeyguardManager>()
+        if (manager?.isKeyguardSecure == true) {
+            showPinVerification(manager)
+        } else {
+            // Warn that the device has not been secured
+            if (context.settings().shouldShowSecurityPinWarning) {
+                showPinDialogWarning(context)
+            } else {
+                promptsFeature.get()!!.onActivityResult(
+                    requestCode = PIN_REQUEST,
+                    data = null,
+                    resultCode = Activity.RESULT_OK
+                )
+            }
```

Comment:
It think we should use instead `promptsFeature.get()?.onBiometricResult( isAuthenticated = true)`